using-mobile-extensions/adobe-campaign-standard/adobe-campaign-standard-event-reference.md

```diff
@@ -165,11 +165,15 @@ Here is the definition of the key-value pairs in this event:
 | **Key** | **Value Type** | **Optional** | **Description** |
 | :--- | :--- | :--- | :--- |
 | `a.message.id` | String | No | Message ID of the message that was triggered. |
-| `a.message.triggered` | String \("1"\) | No | Flag that specifies the type of message interaction event. |
+| `a.message.triggered` | String \("1"\) | No | Flag that specifies a message triggered event. Note, only one type of message interaction is present in each campaign response content event. |
```

Comment:
- In the "Description" column, please remove "Note," and start the sentence with "Only". 
- Please replace "is present" with "exists" and capitalize "campaign".